db/migrate/20230923135510_create_tasks.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTasks < ActiveRecord::Migration[6.1]
+  def change
+    create_table :tasks do |t|
+      t.string :name, null: false
+      t.text :description
+      t.string :status, null: false
+      t.datetime :deadline_at
```

Comment:
After you change something in migrations, you need to execute it, for apply changes in schema, now schema has a `t.datetime "deadline"` field as before